samples/browser/src/main/java/org/mozilla/samples/browser/BrowserActivity.kt

```diff
@@ -43,6 +53,32 @@ open class BrowserActivity : AppCompatActivity(), ComponentCallbacks2 {
         }
     }
 
+    @kotlinx.coroutines.ExperimentalCoroutinesApi
+    override fun onStart() {
+        super.onStart()
+        scope = components.store.flowScoped { flow ->
+            flow.ifChanged { it.extensions }
+                .map { it.extensions.filterValues { extension -> extension.popupSession != null } }
+                .collect { pair ->
```

Comment:
TODO rename; this is not a pair...

Reply:
Yeah, maybe just extensionState?

Reply:
nit: I would (also for Fenix) move this to a private function to make it clear what this does

```
webExtScope = observeWebExtensionPopups()
```

Reply:
I like it